NodeMCU/src/Client/client.ino

```diff
@@ -4,22 +4,36 @@
 void setup() { 
   pinMode(D5, OUTPUT);
   Serial.begin(115200);                                  //Serial connection
-  WiFi.begin("RonaZona", "squidward420");   //WiFi connection
+  WiFi.begin("RonaZona", "squidward420");                //WiFi connection
   delay(5000);
 }
  
 void loop() {
- if(WiFi.status() == WL_CONNECTED){   //Check WiFi connection status
-   HTTPClient http; //Declare object of class HTTPClient
+ if(WiFi.status() == WL_CONNECTED){                      //Check WiFi connection status
+   HTTPClient http;                                      //Declare object of class HTTPClient
```

Comment:
I would suggest adding more HTTPClient variables to account for our different components and to potentially help eliminate server request noise.